core/lib/json_api_client_extension/has_one_extension.rb

```diff
@@ -0,0 +1,27 @@
+module JsonApiClientExtension::HasOneExtension
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def has_one(attr_name, options = {})
+      class_eval <<-CODE
+        def #{attr_name}_id=(id)
```

Comment:
Coud you add something like:
```ruby
ActiveSupport::Deprecation.warn('Use relationships instead')
# => "DEPRECATION WARNING: Use relationships instead (called from your_code.rb:1)"
```

This will allow us to spot and progressively replace all the calls

Reply:
👍